examples/vic_basic/vic_basic.ino

```diff
@@ -47,25 +53,17 @@ void setup()
 
     vic_var_set("state", "0");
     vic_var_bind("state", &led_state, VIC_VAR_UINT8);
-
-    Serial.println(F("Try to use these commands: (type them and hit enter)"));
-    Serial.println(F("\ton\n\toff"));
-    Serial.println(F("\tled on\n\tled off"));
-    Serial.println(F("\tset state 1\n\tset state 0"));
-    Serial.println(F("and see LED attached to pin 13"));
-    Serial.println(F("If you want to try more:\n\tls\n\tlsv"));
-
-    Serial.println(F(VIC_PS1));
 }
 
 void loop()
 {
     if (Serial.available() > 0) {
         char c = Serial.read();
+
+        /* to delete character in picocom */
+        if (c == '\b') Serial.print(" \b");
```

Comment:
@vargac Just out of curiosity, why is the space necessary? In other words, I understand the `\b` but I do not quite get ` \b`.